src/NuGet.Clients/NuGet.VisualStudio/Extensibility/IVsPackageInstallerServices.cs

```diff
@@ -13,7 +13,6 @@ namespace NuGet.VisualStudio
     /// </summary>
     [ComImport]
     [Guid("B858E847-4920-4313-9D3B-176BB0D2F5C2")]
-    [Obsolete("Use INuGetProjectService in the NuGet.VisualStudio.Contracts package instead.")]
```

Comment:
This probably shouldn't get removed. Likely a consequence of a bad merge.

Reply:
Oh, thanks for catching this. I had to remove this for the recent insertion because there is code in VS that is still using it and the ObsoleteAttribute broke the build. So investigating those two usages is another thing I'll need to take care of before completing this PR.

Reply:
Reverted this change.